python/neuroglancer/ingest/tasks.py

```diff
@@ -129,25 +415,44 @@ def update(self, task):
         """
         raise NotImplemented
 
-    def lease(self):
+    def lease(self, tag=''):
         """
         Acquires a lease on the topmost N unowned tasks in the specified queue.
         Required query parameters: leaseSecs, numTasks
         """
-        tasks = self.api.lease(
-            project=self._project,
-            taskqueue=self._queue_name, 
-            numTasks=1, 
-            leaseSecs=600).execute()
+        if not tag:
```

Comment:
It all depends on what is in the queue.
You can have for example dowsample, superdowsample, and bigarray tags and have a worker indiscriminately processing any of them.
But if we have an inference task that requieres the worker to have a gpu, you would want that worker to only accept inference tasks.